src/plugins/intel_npu/src/backend/include/zero_executor.hpp

```diff
@@ -35,18 +36,12 @@ class ZeroExecutor final : public IExecutor {
     };
 
     void setArgumentValue(uint32_t argi_, const void* argv_) const;
-    void setWorkloadType(const ov::WorkloadType workloadType) const override;
+    void setWorkloadType(ov::WorkloadType workloadType) const override;
     void mutexLock() const;
     void mutexUnlock() const;
     inline ze_graph_handle_t graph() const {
         return _graph;
     }
-    inline std::shared_ptr<const ZeroInitStructsHolder> getInitStructs() const {
-        return _initStructs;
-    }
-    inline const std::shared_ptr<const NetworkDescription>& getNetworkDesc() const {
-        return _networkDesc;
-    }
     inline const std::shared_ptr<CommandQueue>& getCommandQueue() const {
```

Comment:
It's outside of PR scope, but just for the future maybe - no sense to write `inline` for member-functions, they are implicitly `inline` by the standard:

> A function defined within a class definition is an inline function.